docs/guides/6to7guide.md

```diff
@@ -0,0 +1,157 @@
+---
+title: restify 6.x to 7.x migration guide
+permalink: /docs/6to7/
+---
+
+## Introduction
+
+restify `7.x` comes with a completely new router and middleware logic that
+brings significant performance improvement to your application.
+From `v7.0.0` restify uses the Radix Tree based
+[find-my-way](https://github.com/delvedor/find-my-way) package as a router
+backend.
+
+## Breaking Changes
+
+### Server throws `RequestCloseError` instead of `RequestAbortedError`
```

Comment:
I don't think the server actually "throws" right? Do we mean that the server just logs a `RequestCloseError`? We may also want to update RequestCloseError's default status code to 444.